ownCloud/Client/Actions/Actions+Extensions/CreateFolderAction.swift

```diff
@@ -22,7 +22,7 @@ class CreateFolderAction : Action {
 	override class var identifier : OCExtensionIdentifier? { return OCExtensionIdentifier("com.owncloud.action.createFolder") }
 	override class var category : ActionCategory? { return .normal }
 	override class var name : String? { return "Create Folder".localized }
-	override class var locations : [OCExtensionLocationIdentifier]? { return [ .plusButton ] }
+	override class var locations : [OCExtensionLocationIdentifier]? { return [ .folderAction ] }
```

Comment:
Please check capitalization of "Create Folder". I only found "Create folder" in localizable.strings file

Reply:
Good catch! Fixed.